man/sd_listen_fds.xml

```diff
@@ -198,6 +198,9 @@
     number. <function>sd_listen_fds_with_names()</function> does the
     same but also parses <varname>$LISTEN_FDNAMES</varname> if
     set.</para>
+
+  <para>Services that specify <varname>StandardInput=socket</varname> are not meant to use
```

Comment:
i'd turn this around: say "these functions are not designed for operation within services that specify StanardInput=socket, as they do not have the $LISTEN_FDS variable set."

or so

Reply:
inverted.